File: jupyter2/problems/conditional/leap_year/jl/leap_year.jl
### begin my answer

### end my answer

function main()
    @assert  is_leap_year(2000)
    @assert !is_leap_year(1900)
    @assert  is_leap_year(2024)
    @assert !is_leap_year(2023)
    println("OK")
end

main()

File: jupyter2/problems/conditional/leap_year/go/leap_year.go
package main
import "fmt"

/** begin my answer */

/** end my answer */
func main() {
	if !isLeapYear(2000) { panic("wrong") }
	if  isLeapYear(1900) { panic("wrong") }
	if !isLeapYear(2024) { panic("wrong") }
	if  isLeapYear(2023) { panic("wrong") }
	fmt.Println("OK")
}
